upup/models/cloudup/resources/addons/aws-load-balancer-controller.addons.k8s.io/k8s-1.9.yaml.template

```diff
@@ -801,6 +801,7 @@ spec:
         - --enable-wafv2=false
         - --enable-shield=false
         - --ingress-class=alb
+        - "--default-tags={{ CsiExtraTags }}"
```

Comment:
can we rename template function to something more generic?

Reply:
hehe. yep. done